features/most-popular-project.js

```diff
@@ -76,6 +76,7 @@ function addBox(title, id, views, instructions) {
   thumbnail.style.width = "300px";
   p.style.textAlign = "left";
   p.style.float = "left";
+  p.style.whiteSpace = "pre";
```

Comment:
This should be `pre-line`, not just `pre`. Otherwise, the text will spill over outside of where it should be.

Reply:
:+1: